src/cmds/scripts/pbs_habitat.in

```diff
@@ -318,6 +318,16 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 			exit $ret
 		fi
 		server_started=1
+		
+		# Check if PBS_SERVER_INSTANCES is set for multi-server
+		if [ ! -z "${PBS_SERVER_INSTANCES}" ]; then
+			# We don't want qmgr to try to contact other servers, which may not even be up yet
+			# So, set PBS_SERVER_INSTANCES to the current server's hostname and port
+			if [ -z "${PBS_BATCH_SERVICE_PORT}" ]; then
+				PBS_BATCH_SERVICE_PORT=15001
+			fi
+			export PBS_SERVER_INSTANCES="$(hostname -s):${PBS_BATCH_SERVICE_PORT}"
```

Comment:
Does this export will get reflected in pbs_init.d and pbs_dataservice where you need to also mask PSI?

Reply:
Those scripts need similar changes in order for the init script to work properly.

Reply:
ok, I'll do that, thanks